buildkitd/entrypoint.sh

```diff
@@ -32,8 +39,16 @@ if [ "$EARTHLY_RESET_TMP_DIR" = "true" ]; then
 fi
 
 if ! lsmod | grep -wq "^ip_tables"; then
-  echo "Legacy ip_tables is not loaded. Switching to iptables-nft."
-  ln -sf /sbin/iptables-nft /sbin/iptables
+  echo "Legacy ip_tables is not loaded."
+
+  if [ $KERNEL = "WSL" ]; then
```

Comment:
just in case $KERNEL ever contains a space:

```suggestion
  if [ "$KERNEL" = "WSL" ]; then
```

Reply:
I hope it won't since we control it; but more security is nice